elliottlib/bzutil.py

```diff
@@ -287,7 +288,7 @@ def is_cve_tracker(bug_obj):
     :param bug_obj: bug object
     :returns: True if the bug is a CVE tracker.
     """
-    return "SecurityTracking" in bug_obj.keywords and "Security" in bug_obj.keywords
+    return "SecurityTracking" in bug_obj.keywords or "Security" in bug_obj.keywords
```

Comment:
IIRC a tracker bug must have both `SecurityTracking` and `Security` keywords. A bug only having `Security` keyword could be a flaw bug (if it is assigned to `Security Response` product). @sfowl Could you provide more info?

Reply:
@vfreex you're understanding is correct.